source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,188 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah SLA AGM battery, capable of high momentary currents.
+
+SLA stands for Sealed Lead Acid, AGM stands for Absorbent Glass Mat.
+
+The Robot Battery assembly includes a COTS battery, with positive and negative lead cables to an Anderson SB connector.
+The Red/Pink SB50 is STRONGLY recommended.
+
+Always start your match with a fully charged battery - ask for help if it looks like you won't!
+
+Battery Safety & Handling
+-------------------------
+
+Battery assemblies are **delicate**, even though they don't look like it.
+
+**DO NOT CARRY A BATTERY BY THE CABLES**, and always avoid pulling by them.
+Pulling on batteries by the cables may loosen, bend, or break the lug or battery tab -- or even tear the entire tab out of the housing!
+
+.. image:: images/robot-battery/broken-terminal.png
+  :alt: One terminal of an FRC battery fully detached from the battery.
+
+Even if it doesn't get broken, the battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+//TODO is this a good place to talk about locking down the SB50 so you don't pull on wire, or is that handled on Common troubleshooting?
```

Comment:
We don't discuss why it is done anywhere but the battery holder is part of the standard kit.  To me this is self explanatory.

Reply:
Added it to "common troubleshooting"